WireUI/Sources/WireIndividualToTeamMigrationUI/Views/CancellationSheet.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireFoundation
+
+@MainActor
+internal func cancellationSheetFactory(
+    onLeave: @escaping @MainActor @Sendable () -> Void,
+    onContinue: @escaping @MainActor @Sendable () -> Void
+) -> UIAlertController {
+    let alert = UIAlertController(
+        title: String.localized(key: "individualToTeam.cancellation.title", bundle: .module),
+        message: String.localized(key: "individualToTeam.cancellation.body", bundle: .module),
+        preferredStyle: .actionSheet
+    )
+    alert.addAction(UIAlertAction(
+        title: String.localized(key: "individualToTeam.cancellation.leave", bundle: .module),
+        style: .destructive,
+        handler: { _ in onLeave() }
+    ))
+    alert.addAction(UIAlertAction(
+        title: String.localized(key: "individualToTeam.cancellation.continue", bundle: .module),
+        style: .cancel,
+        handler: { _ in onContinue() }
+    ))
+    alert.view.tintColor = UIColor(.primary)
```

Comment:
**maybe** we don't want this to be .primary but the annoying user selectable accent color / profile color. Maybe double check. Same comment if there are similar cases elsewhere.

Reply:
No. It's for the bottom sheet title and content. They aren't an accent color; they are gray.

![Screenshot 2024-12-03 at 17 38 02](https://github.com/user-attachments/assets/c97d667d-e9c8-489b-a50a-f8e372d37299)

Reply:
Of course (🤦 maybe I shouldn't review at the end of the day!)

Perhaps the color should be the one we define in `WireDesign` - `Color.primaryText`. As far as I'm aware we were trying to avoid using apples named colors. But perhaps I'm wrong.

Reply:
Yes. I recall there was an issue with customising the color of the title or the body, so I just matched the other one. I’ll have a look and let you know.